python/kfserving/kfserving/protocol.py

```diff
@@ -0,0 +1,4 @@
+class ProtocolHandler(object):
```

Comment:
Can we call this protocol_handler.py?

Reply:
Oh I see now. I see, this class handles both. I think of the protocol as the thing that responds with which protocol we support.

Reply:
Can we put them all in a folder then?

kfserving/protocols/seldon_http.py
kfserving/protocols/tensorflow_http.py

Reply:
Will move. And change name to `RequestHandler` ?